code/datums/elements/food/processable.dm

```diff
@@ -1,4 +1,4 @@
-// If an item has the processable item, it can be processed into another item with a specific tool. This adds generic behavior for those actions to make it easier to set-up generically.
+// If an item has the processable item, it can be processed into another item with a specific tool. This adds generic behaviour for those actions to make it easier to set-up generically.
```

Comment:
there isnt a standard on english v american spelling so im going with the brits because thats how it it is called by every other variable on this document